src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+}
+
+section {
+  box-sizing: border-box;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+    margin-left: 0;
+    margin-bottom: 0;
+  }
+}
+
+.stars__star {
```

Comment:
these styles should be in file with styles for the star block

Reply:
i thought if this is resets settings it had to be at "reset.scss" file, isnt it?

Reply:
in reset.scss we can reset default browser styles for needed tags (using tag selectors)
but if we reset something for elements using selectors by classes, we should add it to the specified css file with corresponding bem block